turbinia/lib/utils.py

```diff
@@ -171,6 +171,7 @@ def bruteforce_password_hashes(
 
   pot_file = os.path.join((tmp_dir or tempfile.gettempdir()), 'hashcat.pot')
   password_list_file_path = os.path.expanduser('~/password.lst')
+  password_rules_file_path = os.path.expanduser('~/turbinia.rules')
```

Comment:
Minor thing, but could you rename this to something slightly more descriptive (e.g. `turbinia-password-cracking.rules`) so it's more obvious what this is for someone that is just looking at files in the home dir?  This is partially because the file contents don't help to distinguish what this is for :).

Reply:
Done